app/Http/Controllers/MemberController.php

```diff
@@ -52,10 +52,19 @@ public function index(Request $request)
      */
     public function store(Request $request)
     {
+        $rules = [
+            'name' => 'required|unique:member|regex:/^[a-zA-Z0-9-. ]+$/u|max:50',
+            'dob' => 'required',
```

Comment:
There are more rules for the member dob: "Required, valid date, not in the future, not older than 60 years"